distributionviewer/core/static/js/app/components/containers/chart-list-container.js

```diff
@@ -12,7 +12,7 @@ export class ChartListContainer extends React.Component {
     this.state = {numCharts: 0};
   }
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
If we would prefer to leave stylistic things like camel-case detection out of eslint entirely, that's fine too.